en/lessons/ecto/basics.md

```diff
@@ -1,5 +1,5 @@
 ---
-version: 2.2.0
+version: 2.2.1
```

Comment:
It looks like we need to apply this change to a number of translation so maybe this should justify `2.3.0`?

Reply:
@doomspork Thank you for your review :)
I changed version 👍 
https://github.com/elixirschool/elixirschool/pull/1923/commits/193ba4e18acc77e90d2b08238e47228a16b7cfdb